client/ayon_flame/plugins/publish/collect_timeline_instances.py

```diff
@@ -209,13 +209,21 @@ def _get_comment_attributes(self, segment):
                 "pixelRatio": 1.00}
         }
         # search for `:`
+        # INFO: clip based overrides needs to have specific format
+        #      `key: value` separated `,` or `;`. This is for cases where we
+        #      need to override resolution, aspect ratio, etc. per clip.
         for split in self._split_comments(comment):
+            self.log.debug(f"__ split: {split}")
+
             # make sure we ignore if not `:` in key
-            if ":" not in split:
+            # of if there is more than one `:` in key
+            if (
+                ":" not in split
+                or split.count(":") > 1
+            ):
```

Comment:
```suggestion
            if split.count(":") != 1:
```

Reply:
Why not used this suggestion? it is simpler, and result is same.

Reply:
it is implemented in the code via different commit